boards/arm/nrf9160_pca20035/nrf9160_pca20035_common.dts

```diff
@@ -172,9 +172,14 @@
 	cts-pin = <25>;
 };
 
-/* UART2 must be enabled for defines to be available for SPM. */
+/*
+ * UART2 must be enabled for defines to be available for SPM for now.
+ * "Required" pins are set to the disconnect value 0xFFFFFFFF.
```

Comment:
Is it correct to use 0xFFFFFFFF for pin number? I dont see such value used for other peripherals using GPIO pins. Set a default recommended pins.

Reply:
UART2 is not used. Writing 1 to the MSB of the PSEL registers of the peripheral tells it that this pin is disconnected.

Reply:
Also, there are no free pins on that board it could point to. The only reason UART2 is present in the DTS is to satisfy the SPM.

Reply:
In other places, I have opted to add the `uart2` node to the project DTS overlay instead of the board, so that the board DTS file reflects the actual hardware. Perhaps a similar solution could be used here.

Reply:
Ok, I moved the workaround to the SPM